lib/Twig/Extensions/Extension/I18n.php

```diff
@@ -17,7 +28,7 @@ class Twig_Extensions_Extension_I18n extends Twig_Extension
      */
     public function getTokenParsers()
     {
-        return array(new Twig_Extensions_TokenParser_Trans());
+        return array(new Twig_Extensions_TokenParser_Trans($singularFunc = $this->singularFunc, $pluralFunc = $this->pluralFunc));
```

Comment:
On 03/03/16 12:13, Christophe Coevoet wrote:
> In lib/Twig/Extensions/Extension/I18n.php
> <https://github.com/twigphp/Twig-extensions/pull/166#discussion_r54866322>:
>
>> @@ -17,7 +28,7 @@ class Twig_Extensions_Extension_I18n extends Twig_Extension
>>       */
>>      public function getTokenParsers()
>>      {
>> -        return array(new Twig_Extensions_TokenParser_Trans());
>> +        return array(new Twig_Extensions_TokenParser_Trans($singularFunc = $this->singularFunc, $pluralFunc = $this->pluralFunc));
>
> why are you assigning local variables here ?

Because I've lately been primarily programming in Python and don't know 
"modern" php coding culture very well.


-- 
HM